src/foam/nanos/auth/LanguageChoiceView.js

```diff
@@ -183,7 +186,7 @@ foam.CLASS({
       let country = await this.countryDAO.find(language.variant)
       let label = language.variant != "" ? `${language.name}(${language.variant})` : `${language.name}`
       if ( country && country.name != null ) {
-        label = `${language.name}(${this.getCountryName(country.name, country.name)})`
+        label = `${language.name}\u00A0(${this.getCountryName(country.name, country.name)})`
```

Comment:
Does this need unicode? A backtick should already detect a space.

Reply:
Xuerong had suggested that but it caused the string to break and overlap. After looking online I found that unicode and it worked fine.